Assets/Engine/Interactions/IntentManager.cs

```diff
@@ -19,13 +19,27 @@ public class IntentManager : MonoBehaviour
     public Color colorHarm;
     public Color colorHelp;
 
+    private Button IntentButton;
+
     public enum IntentType
     {
         Help,
         Harm
     }
+
+    void Start()
+    {
+        IntentButton = GetComponent<Button>();
+        IntentButton.onClick.AddListener(TaskOnClick);
+    }
+
+    void TaskOnClick()
+		{
+			SelectIntent();
+		}
+
     // Select the body part and deselect others
```

Comment:
I know this was already in the script before you touched it, but it'd be nice to get rid of this comment. It's just a copy-paste from another script.